### DISCUSSION TITLE
Version 0.0.* and no output directory

### DISCUSSION THREAD

**Discussion by CB9683** [HALFpipe/HALFpipe#623] [Q&A]
Hi Lea, 
Thanks for this crazy cool tool! I used it yesterday for the first time. Here are two things I was wondering about:
I downloaded the container for Singularity 3.x. I let it run on an HPC and it goes through without errors. But, an output directory was not generated, the results are still in the working directory.  I was interested in the atlas-based connectivity, but I am not sure, if the results are present. I attached the tree of my workdir and the log file. The job seems to end abruptly, but there was no error message generated from the job scheduler.
[log.txt](https://github.com/HALFpipe/HALFpipe/files/12889971/log.txt)
[tree.txt](https://github.com/HALFpipe/HALFpipe/files/12889972/tree.txt)
Furthermore I was wondering, that I was working with version 0.0.* (see screenshot). That seems a bit odd, because I asked for the latest version.
![Screenshot 2023-10-12 134552](https://github.com/HALFpipe/HALFpipe/assets/133920974/dc44f216-c74e-46a3-849a-ba4c9699f730)
Thanks for your help and keep up the fantastic work,
Christian

**Comment by HippocampusGirl** [comment]
Hi Christian,

Thank you for reporting this issue! I believe this means something is going wrong during the build process, as the version number is not being embedded correctly. 

To work around the issue for now, could you try the current release (version 1.2.2)? That should always work as expected.

Best,
Lea
